mysite/scrapers/scraperDomRiaScraper.py

```diff
@@ -1,22 +1,107 @@
+
+    
+    def scrape_property_details(self) -> Dict:
+
+        response = self.get_page()
+        if not response:
+            return {}
+        
+
+        soup = BeautifulSoup(response.text, "html.parser")
+        
+        property_details = {
+            'url': self.listing_url,
+            'source_website': 'dom.ria.com',
+            'created_at': datetime.datetime.now(),
+            'last_checked_at': datetime.datetime.now()
+        }
+
         deleted_tag = soup.find("span", class_="size24 bold")
         if deleted_tag and "видалено" in deleted_tag.text.lower():
-            availability = "deleted"
-            normalized_price = None
+            property_details['availability'] = 'deleted'
+
         else:
-            availability = "available"
+            property_details['availability'] = 'available'
             price_tag = soup.find("b", class_="size30")
             raw_price = price_tag.text.strip() if price_tag else None
             normalized_price = self.normalize_price(raw_price)
-        return {
-            "url": self.website_url,
-            "price": normalized_price,
-            "availability": availability
-        }
+
+            if normalized_price is None:
+                cleaned_price = "0"
+            else:
+                cleaned_price = re.sub(r'\D', '', normalized_price)
+
+            try:
+                property_details['original_price'] = int(cleaned_price)
+            except ValueError:
+                property_details['original_price'] = 0
+
+            
+            description_elem = soup.select_one("div#mainDescription")
+            if description_elem:
+                property_details['description'] = description_elem.text.strip()
+
+            detail_rows = soup.select("ul.main-list li")
+            for row in detail_rows:
+                text = row.get_text(" ", strip=True)
+
+                if "кімнат" in text:
+                    rooms_match = re.search(r'(\d+)', text)
+                    if rooms_match:
+                        try:
+                            property_details['number_of_rooms'] = int(rooms_match.group(1))
+                        except ValueError:
+                            pass
+            
+                if "поверх" in text:
+                    floor_match = re.search(r'(\d+)\s*поверх', text)
+                    if floor_match:
+                        try:
+                            property_details['floor'] = int(floor_match.group(1))
+                        except ValueError:
+                            pass
+                
+                if 'Загальна площа' in text:
+                    area_match = re.search(r'Загальна площа\s*(\d+(?:\.\d+)?)', text)
+                    if area_match:
+                        try:
+                            property_details['total_area'] = float(area_match.group(1))
+                        except ValueError:
+                            pass
+
+        print(property_details)
+
```

Comment:
Please try to keep the spacing consistent. I would suggest to use the existing [Style Guide for Python Code](https://peps.python.org/pep-0008/#blank-lines) (although I will double check if I myself if following it)